src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
+      headerBackTitle,
+      headerTitleStyle,
+    } = this.props.getScreenDetails(lastScene).options;
+
+    if (headerBackTitle === null) {
```

Comment:
My idea was that there's no use in forwarding the style if the title isn't there anyway. But I can remove the check.